__init__.py

```diff
@@ -267,10 +261,12 @@ def handle_light_set_intent(self, message):
 
         # IDEA: set context for 'turn it off again' or similar
         # self.set_context('Entity', ha_entity['dev_name'])
-
+        # Set values for HA
         ha_data['brightness'] = brightness_value
+        self.ha.execute_service("light", "turn_on", ha_data)
+        # Set values for mycroft reply
         ha_data['dev_name'] = ha_entity['dev_name']
-        self.ha.execute_service("homeassistant", "turn_on", ha_data)
```

Comment:
I can't test this so I ask for confirm. Is light the right service to call? The old homeassistant service call is not working?

Reply:
yes, `light.turn_on` accept `brightness` settings, `homeassistant.turn_on` just `On/Off`